packages/README.md

```diff
@@ -41,8 +41,8 @@ this or not, but in the interest of keeping it easy, we want to lean towards usi
 decisions between the two of them for "directory layout", "which tools run repo-wide vs
 per-package", etc.
 
-## Current (Oct 2020) package layout
+## Current (July 2021) package layout
 
-* `/src/packages/accessibility-insights-ui/root`: current home of future `/packages/ui`
-* `/src/reports/package/root`: current home of future `/packages/report`
+* `/packages/ui`: current home of former `/src/packages/accessibility-insights-ui/root`
+* `/packages/report`: current home of former `/src/reports/package/root`
```

Comment:
Since the contents of these `src/**/**/root` directories have been already made, should we remove these notes? I left it in and slightly modified the text because it is titled "Current package layout" instead of something like "Changes needed to be made"

Reply:
I think removing the notes makes sense!

Reply:
On second reading, maybe keeping the section makes sense (since it also talks about where to find the web/unified packages outside the directory, which isn't obvious). I'm not sure we need to note the "of former" locations, but it might make sense to note a one-sentence summary of what each folder represents.

Reply:
There seems to already be a description for each of these folders under "Short-term (~Dec 2020) planned package layout" in the same README. Would it make sense to have a similar summary here as well?